custom-metrics-stackdriver-adapter/examples/prometheus-to-sd/README.md

```diff
@@ -41,25 +54,30 @@ env:
       fieldRef:
         fieldPath: metadata.namespace
 ```
-## Horizontal Pod Autoscaling object
 
-In the example, there is a [Horizontal Pod Autoscaler object] configured to scale the deployment on 
-the exported metric. It is configured with a target value lower than the defaul metric value. This will
-cause the deployment to scale up to 5 replicas over the course of ~15 minutes. 
-If you do not see a successful scale up event for a longer period of time, there is 
-likely an issue with your setup. See [Troubleshooting](#troubleshoooting) section for possible reasons.
 
-## Deployment
+### Option 2. Exporting metrics to Google Cloud Managed Service for Prometheus
 
-The example can be used to test your custom metrics setup using Stackdriver. See the [custom metrics setup] instructions for more details.
+Alternatively, you can use [Google Cloud Managed Service for Prometheus](https://cloud.google.com/managed-prometheus) (GMP) to manage your metrics.
+
+#### Background
+
+If you would like to learn more about GMP, you can check out the [documentation](https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-managed) to get yourself familair with it.
```

Comment:
Please use Google Cloud Managed Service for Prometheus instead of GMP anywhere in the doc.

Typo: familiar